README.md

```diff
@@ -249,6 +249,15 @@ ReactDOM.render(<App />, document.getElementById('app'));
 
 This component uses [BEM](http://csswizardry.com/2013/01/mindbemding-getting-your-head-round-bem-syntax/) for namespacing the CSS classes. So styling should be easy and without conflicts. See the [geosuggest.css](https://github.com/ubilabs/react-geosuggest/blob/master/src/geosuggest.css) for an example styling.
 
+### Note:
+`geosuggest__suggests--hidden` class is added to auto hide the suggestion list. copy the below style in your CSS file.
```

Comment:
Please use a capital letter on `Copy`. And can you add why one should add this CSS? It's clear to us now. But new ones might not get why this could be important.